src/helpers/strings.ts

```diff
@@ -14,8 +14,9 @@ export class Strings {
     static NavigateToBuildSummary: string = "Click to view build";
     static NavigateToTeamServicesWebSite: string = "Click to view your team project website.";
     static NoAccessTokenFound: string = "A personal access token for this Team Services repository was not found in your local user settings.";
-    static NoAccessTokenRunLogin: string = "No personal access token was found for Team Services.  Please run the 'team login' command.";
-    static NoTeamServerCredentialsRunLogin: string = "No credentials were found for Team Foundation Server.  Please run the 'team login' command.";
+    static NoAccessTokenLearnMoreRunLogin: string = "You are not connected to Team Services. Please learn more and then run the 'team login' command.";
```

Comment:
@DavidStaheli  :)  @AlexRukhlin made the case to use "Please learn more" as either "selecting" or "clicking" could mean (to the user) that all they have to do is click the button, not do anything else, and then run the command.  He also made the point that the buttons are hard to miss.  I could go either way.  Thoughts?